x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/AbstractEsqlIntegTestCase.java

```diff
@@ -39,6 +43,19 @@ public void ensureExchangesAreReleased() throws Exception {
         }
     }
 
+    @After
+    public void ensureBlocksReleased() {
+        for (String node : internalCluster().getNodeNames()) {
+            CircuitBreakerService breakerService = internalCluster().getInstance(CircuitBreakerService.class, node);
+            CircuitBreaker reqBreaker = breakerService.getBreaker(CircuitBreaker.REQUEST);
+            try {
+                assertBusy(() -> assertThat("Request breaker not reset to 0 on node: " + node, reqBreaker.getUsed(), equalTo(0L)));
+            } catch (Exception e) {
+                assertThat("Request breaker not reset to 0 on node: " + node, reqBreaker.getUsed(), equalTo(0L));
+            }
+        }
+    }
```

Comment:
ESIntegTest case already checks this, doesn't it?

Reply:
Oh! I see. You can call it any time  you like. That's nice. I was thinking of having one of this. Maybe it just doesn't need to be `@After` because we already check the breaker in another spot.

Reply:
I removed `After` in https://github.com/elastic/elasticsearch/pull/100295/commits/b2d39c2699178d1db902fe3f909e472e1076cb60